web/modules/custom/joinup_news/joinup_news.module

```diff
@@ -22,3 +31,123 @@ function joinup_news_entity_bundle_field_info_alter(&$fields, EntityTypeInterfac
     }
   }
 }
+
+/**
+ * Implements hook_node_access().
+ *
+ * Interrupts the permissions for updating a node. The permission is not
+ * granted if there are no transitions allowed for the current state of the
+ * node.
+ */
+function joinup_news_node_access(EntityInterface $node, $op, AccountInterface $account) {
+  if ($op == 'update' && $node->bundle() == 'news') {
```

Comment:
You could check earlier if it's not a news bundle and return immediately `AccessResult::neutral()`. Personal preference.